web-app/src/pages/Dashboard/container.js

```diff
@@ -72,7 +73,9 @@ const DashboardContainer = Wrapped =>
           start: new moment(hol.date, 'YYYY-MM-DD'),
           end: new moment(hol.date, 'YYYY-MM-DD'),
           isHalfday: hol.halfDay,
+          isWFH: false,
           ...hol,
+          holidayStatusId: Math.random() < 0.3 ? 4 : hol.holidayStatusId,
```

Comment:
I'm not quite sure what this is about? Basically this has a 3/10 chance of being WFH?

Reply:
Yep! Since the backend doesn't support remote working events yet I added this so a few events on the calendar would be remote. Again, just for demonstration.